{"name": "test_bas_result_check", "status": "failed", "statusDetails": {"message": "AssertionError: assert True == False\n  +True\n  -False", "trace": "test_bad_result.py:3: in test_bas_result_check\n    assert True == False\nE   assert True == False\nE     +True\nE     -False"}, "attachments": [{"name": "log", "source": "3293e0a2-d1e3-4ef2-af73-3f49977203bb-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "ebc01c0c-4cdb-459b-9e88-cd4c257a12bc-attachment.txt", "type": "text/plain"}], "start": 1596530397928, "stop": 1596530397928, "uuid": "9030d7e2-209d-4d6a-8b1d-a0e1de2d84e8", "historyId": "b90642ceb2f3b4a90b5166f617bed14a", "testCaseId": "901d9abe59f0ff217281fd5a08fe93da", "fullName": "test_bad_result#test_bas_result_check", "labels": [{"name": "suite", "value": "test_bad_result"}, {"name": "host", "value": "S340"}, {"name": "thread", "value": "31115-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_bad_result"}]}